exchanges/kraken/kraken_types.go

```diff
@@ -545,23 +538,11 @@ type WebsocketSubscriptionResponseData struct {
 	Name string `json:"name"`
 }
 
-// WebsocketDataResponse defines a websocket data type
-type WebsocketDataResponse []interface{}
-
 // WebsocketErrorResponse defines a websocket error response
 type WebsocketErrorResponse struct {
 	ErrorMessage string `json:"errorMessage"`
 }
 
-// WebsocketChannelData Holds relevant data for channels to identify what we're
-// doing
-type WebsocketChannelData struct {
```

Comment:
Whilst you're cleaning these up, please also remove `wsSubscription` as its unusued https://github.com/gbjk/gocryptotrader/blob/feature/kraken_ws_v5/exchanges/kraken/kraken_types.go#L559

Reply:
Fixed 10d6a19